### PR TITLE
merge FSharp.Core nuget package

### PR DESCRIPTION
**Pull request by dsyme**
This makes the FSharp.Core package include a merge of the ``Microsoft.FSharp.Core.netcore.1.0.0-alpha-160627`` package, see also https://github.com/Microsoft/visualfsharp/issues/1054 and https://github.com/fsharp/fsharp/issues/566

This will need to be updated once that package is finalized

### CODE REVIEW

File: packages.config [hunk 1]
```diff
@@ -14,4 +14,6 @@
   <package id="Microsoft.DiaSymReader" version="1.0.8" />
   <package id="FsSrGen" version="3.0.0" targetFramework="net46" />
   <package id="FSharp.SRGen.Build.Tasks" version="3.0.0" targetFramework="net46" />
+  <package id="NupkgMerge" version="1.0.0.1" />
+  <package id="Microsoft.FSharp.Core.netcore" version="1.0.0-alpha-160627" />
```
Comment (enricosada) [inline packages.config]: package is `1.0.0-alpha-160629`, it's not in fsharp-daily feed, but is on nuget.org

**Comment by forki** [comment]
that's a good idea. but the coreclr version has more fixes than the net4x version, right?

**Comment by dsyme** [comment]
@forki Yes, it does (though very few)

**Comment by dsyme** [comment]
BTW I agree this package should be labelled as alpha, I'll unlist it and we can republish

**Pull request merged**